I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script/gym.rpy", line 24, in script
    show aina basic with easein
  File "game/script/gym.rpy", line 24, in <module>
    show aina basic with easein
NameError: name 'easein' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script/gym.rpy", line 24, in script
    show aina basic with easein
  File "C:\Users\evans\Documents\renpy-7.1.3-sdk\renpy\ast.py", line 1352, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\Users\evans\Documents\renpy-7.1.3-sdk\renpy\python.py", line 1944, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\evans\Documents\renpy-7.1.3-sdk\renpy\python.py", line 1937, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script/gym.rpy", line 24, in <module>
    show aina basic with easein
NameError: name 'easein' is not defined

Windows-8-6.2.9200
Ren'Py 7.1.3.1092
Break: The Rematch v0.2.0-alpha
Thu Dec 27 10:00:21 2018
